### PR TITLE
Added more test cases for two rules

### LINKED ISSUE #562: Lint rule htmlHasLang has failing test cases
The original implementation of `htmlHasLang` had diagnostics for these test cases:

    <html lang={false}></html>
    <html lang={true}></html>
    <html lang={42}></html>

But the current implementation does not.

### LINKED ISSUE #563: Lint rule iframeHasTitle has failing test cases
The original implementation of `iframeHasTitle` had diagnostics for these test cases:

    <iframe title={``} />
    <iframe title={false} />
    <iframe title={true} />
    <iframe title={42} />

But the current implementation does not.

### PR DESCRIPTION
**Pull request by ematipico**
Fixes #562 
Fixes #563 

Added more test cases an utility that checks if the template literal is empty

### CODE REVIEW

File: packages/@romejs/js-ast-utils/isEmptyTemplateLiteral.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {JSTemplateLiteral} from "@romejs/ast";
+
+export default function isEmptyTemplateLiteral(node: JSTemplateLiteral) {
+	if (node.quasis.length === 1) {
```
Comment (bitpshr) [inline packages/@romejs/js-ast-utils/isEmptyTemplateLiteral.ts:4]: Should this also be checking if all `quasis` are empty, even if there are more than one?

Comment (ematipico) [reply packages/@romejs/js-ast-utils/isEmptyTemplateLiteral.ts:4]: I checked using https://kevinkelbie.github.io/ (AST explorer with `rome-js`) and with empty `{``}` template literal the `quasis` array is always not empty. But I can add this check anyway :)
